core/base/src/TQObject.cxx

```diff
@@ -1011,7 +1011,7 @@ Bool_t TQObject::Disconnect(const char *class_name,
    TClass *sender = TClass::GetClass(class_name);
 
    // sender should be TQClass (which derives from TQObject)
-   if (!sender->IsA()->InheritsFrom(TQObject::Class()))
+   if (!sender || !sender->IsA()->InheritsFrom(TQObject::Class()))
```

Comment:
Why sender->IsA() used here?
sender is already pointer on the class, one should just check sender->InheritsFrom(TQObject::Class())

Reply:
Well, it's your code...